src/capture/mod.rs

```diff
@@ -13,6 +14,7 @@ pub enum CaptureArgs {
     /// Capture open window information from a (linux) X11 server
     X11(X11CaptureArgs),
     Windows(WindowsCaptureArgs),
+    OSX(OSXCaptureArgs),
```

Comment:
why are you calling it OSX? from what i understand, macos is the official name since a few years ago

Reply:
No particular reason, it was just named as OSX on our todo list in the company and I followed it. We can change it if need be.

Reply:
then please rename it to macos while the refactoring is still easy, that's what most people seem to be calling it

Reply:
No problem, will do.

Reply:
Done.